datasets/xquad_r/xquad_r.py

```diff
@@ -0,0 +1,142 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TODO: Add a description here."""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+# TODO: Add BibTeX citation
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@article{roy2020lareqa,
+  title={LAReQA: Language-agnostic answer retrieval from a multilingual pool},
+  author={Roy, Uma and Constant, Noah and Al-Rfou, Rami and Barua, Aditya and Phillips, Aaron and Yang, Yinfei},
+  journal={arXiv preprint arXiv:2004.05484},
+  year={2020}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+XQuAD-R is a retrieval version of the XQuAD dataset (a cross-lingual extractive QA dataset). Like XQuAD, XQUAD-R is an 11-way parallel dataset, where each question appears in 11 different languages and has 11 parallel correct answers across the languages.
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://github.com/google-research-datasets/lareqa"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://github.com/google-research-datasets/lareqa/raw/master/xquad-r/"
+_LANG = ["ar", "de", "zh", "vi", "en", "es", "hi", "el", "th", "tr", "ru"]
+
+
+class XquadRConfig(datasets.BuilderConfig):
+
+    """ BuilderConfig for XquadR"""
+
+    def __init__(self, lang, **kwargs):
+        """
+        Args:
+            lang: string, language for the input text
+            **kwargs: keyword arguments forwarded to super.
+        """
+        super(XquadRConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+        self.lang = lang
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class XquadR(datasets.GeneratorBasedBuilder):
+    """TODO(xquad-r): Short description of my dataset."""
+
+    # TODO(xquad-r): Set up version.
+    VERSION = datasets.Version("1.1.0")
```

Comment:
```suggestion
    VERSION = datasets.Version("1.1.0")
    BUILDER_CONFIG_CLASS = XquadRConfig
```
we need that to specify that we're using a custom config